runtime/services/catalog/migrations_test.go

```diff
@@ -290,6 +290,7 @@ func TestReconcileMetricsView(t *testing.T) {
 	// dropping the timestamp column gives a different error
 	require.Equal(t, metrics_views.TimestampNotFound, result.Errors[0].Message)
 
+	// some invalid parts still creates an entry in catalog for dashboard APIs to work
```

Comment:
This is a merge conflict right? Please remove the comment.

Reply:
Hmm it doesn't seem to be registering as a merge conflict. I merged in `main` and no conflicts remaining. I removed the comment though.